gnosis/safe-react
# Add hover state to Icons

## What it solves
Closes #2347
## How this PR fixes it
By adding a :hover state for icons (copyBtn, QR, Replace, Edit and Delete)
## How to test it
Open a Safe and check how the icons change the color when the pointer is over the icon (on the sidebar, address book and settings). 
## Any extra information
https://www.loom.com/share/9c5602a59d24437aad3dc04710e73440
